### DISCUSSION TITLE
Restrictions for data object folder

### DISCUSSION THREAD

**Discussion by blankse** [dachcom-digital/pimcore-members#189] [Q&A] [answered]
| Q                | A
| ---------------- | -----
| Bug report?      | no
| Feature request? | yes
| BC Break report? | no
| RFC?             | yes/no

We want to set the restriction to the data object folder. It should be inherited to the objects. Is this possible?

**Comment by solverat** [comment, marked as answer]
Hey @blankse

I'm afraid, but that is not possible (at least, not intended :) ). 

However, it's possible to inherit information to child objects, so it wouldn't be a big thing to introduce this feature on folder layer. The same goes for folders within the document tree.

I've created an issue (https://github.com/dachcom-digital/pimcore-members/issues/190) but I can't say when we will implement this feature.
